### ISSUE TITLE
Metrics error on headless machine

### ISSUE THREAD

**Issue by jfairley** [webpro/release-it#137]
I run builds an Jenkins. Things like window size aren't available in this environment. This error message aborts my release.

```
00:08:17 + ./node_modules/.bin/release-it minor --non-interactive --verbose
00:08:20 /home/ec2-user/workspace/RELEASE/access/node_modules/release-it/lib/metrics.js:50
00:08:20 var dimensions = _windowSize2.default.get();
00:08:20                                      ^
00:08:20 
00:08:20 TypeError: Cannot read property 'get' of undefined
00:08:20     at Object.<anonymous> (/home/ec2-user/workspace/RELEASE/access/node_modules/release-it/lib/metrics.js:15:20)
00:08:20     at Module._compile (module.js:409:26)
00:08:20     at loader (/home/ec2-user/workspace/RELEASE/access/node_modules/release-it/node_modules/babel-register/lib/node.js:144:5)
00:08:20     at Object.require.extensions.(anonymous function) [as .js] (/home/ec2-user/workspace/RELEASE/access/node_modules/release-it/node_modules/babel-register/lib/node.js:154:7)
00:08:20     at Module.load (module.js:343:32)
00:08:20     at Function.Module._load (module.js:300:12)
00:08:20     at Module.require (module.js:353:17)
00:08:20     at require (internal/module.js:12:17)
00:08:20     at Object.<anonymous> (/home/ec2-user/workspace/RELEASE/access/node_modules/release-it/lib/index.js:4:1)
00:08:20     at Module._compile (module.js:409:26)
00:08:20     at loader (/home/ec2-user/workspace/RELEASE/access/node_modules/release-it/node_modules/babel-register/lib/node.js:144:5)
00:08:20     at Object.require.extensions.(anonymous function) [as .js] (/home/ec2-user/workspace/RELEASE/access/node_modules/release-it/node_modules/babel-register/lib/node.js:154:7)
00:08:20     at Module.load (module.js:343:32)
00:08:20     at Function.Module._load (module.js:300:12)
00:08:20     at Module.require (module.js:353:17)
00:08:20     at require (internal/module.js:12:17)
00:08:20 Build step 'Execute shell' marked build as failure
00:08:22 Finished: FAILURE
```

**Comment by webpro** [comment]
Hi @jfairley! Apologies for this, that was pretty bad. I've fixed it in [v4.1.1](https://github.com/webpro/release-it/releases/tag/4.1.1) while also adding some test coverage (runs in travis-ci).

**Comment by jfairley** [comment]
Thanks! I'll give it a try.
